apps/console/src/features/core/configs/app.ts

```diff
@@ -278,6 +278,7 @@ export class Config {
                 defaultWhiteLogoUrl: window[ "AppUtils" ]?.getConfig()?.ui?.emailTemplates?.defaultWhiteLogoUrl
             },
             enableCustomEmailTemplates: window[ "AppUtils" ]?.getConfig()?.ui?.enableCustomEmailTemplates,
+            excludeIdentityClaims: window[ "AppUtils" ]?.getConfig()?.ui?.excludeIdentityClaims,
```

Comment:
If config is undefined, set the value to true by default.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5350/commits/fa3eaa3bd7ef78fc42454367b8cd8d7796438f66